recipes/graphql-ws/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "graphql-ws" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b4e80b6bc50c29cfa27f3d63e8a289c91d3cffb49022a7dbe96dababcf763159
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
just to confirm, do you really need the ``pytest-runner`` to generate this project?

Reply:
Sorry, forgot to mention it. The `conda build .` command failed, and in the traceback I found that it was trying to use `pytest-runner`. Once I added it, the `conda build .` command passed. I think it is because of this hard requirement in `setup_requires`: https://github.com/graphql-python/graphql-ws/blob/523dd1516d8709149a475602042d6a3c5eab5a3d/setup.py#L22

We had something similar in our first recipe published to Conda Forge, and someone suggested us to load it dynamically only when running `python setup.py test`. But as I am not a maintainer of `graphql-ws`, that would be a bit harder to get done soon I guess.

Reply:
Indeed, you will need it. I just wanted to double-check because ``pytest-runner`` as a ``host`` requirement for a pure python project is not that usual let's say :)